Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -3,10 +3,15 @@
 // Refer to the license.txt file included.
 
 #include "Core/HW/Wiimote.h"
+
 #include "Common/ChunkFile.h"
+#include "Common/CommonTypes.h"
+
```

Comment:
We often put newlines between different categories (Common, Core, DiscIO...) when there are many includes, so that it's easier to read.